commands.go

```diff
@@ -73,6 +75,9 @@ func runAgent() {
 	provisionAgent := agent.NewProvisionAgent(hostname, daemon, pfclient)
 	go provisionAgent.Run()
 
+	bootstrapAgent := agent.NewBootstrapAgent(hostname, fullPath, daemon, pfclient)
```

Comment:
If we do it like this, then provision and bootstrap might happen simultaneously (which is not what we want, race condition may occur)

I think the logic of bootstrap should be put inside provision logic itself,
it should happen after provision process is finished

Reply:
Done